dipy/tracking/tests/test_metrics.py

```diff
@@ -8,140 +8,6 @@
 from dipy.tracking import distances as pf
 
 
-def test_downsample():
```

Comment:
same as above. but you need to add a test to check if the deprecation warning appears. Look the following link to see an example : https://github.com/dipy/dipy/pull/1957/files#diff-1f8b2bf0bddd7571f2d196ac846a37c0R106